account_payment_order/models/account_payment_order.py

```diff
@@ -338,6 +339,12 @@ def draft2open(self):
             for payline in order.payment_line_ids:
                 try:
                     payline.draft2open_payment_line_check()
+                    if str2bool(
+                        self.env["ir.config_parameter"]
+                        .sudo()
+                        .get_param("account_payment_order.use_allow_out_payment")
+                    ):
+                        payline._check_bank_allows_out_payments()
```

Comment:
Don't we need to check that `payment_type` is `outbound` here? Because this check does not make sense for a debit order.